tool/site/client/src/services/test-packs/medicationRequests.ts

```diff
@@ -219,6 +218,26 @@ function getMedicationRequestExtensions(row: PrescriptionRow, prescriptionTreatm
     })
   )
 
+  if (controlledDrugQuanitity && controlledDrugSchedule) {
```

Comment:
are we going to account for user error?

Reply:
We could throw an error if one field is present but not both however I'd prefer to do that as a separate sub task ticket and get this in so that we can get it out to suppliers